triangle.json

```diff
@@ -0,0 +1,98 @@
+{
+    "#": [
+        "Pursuant to discussion in #202,",
+        "we have decided NOT to test triangles where a + b = c.",
+        "e.g: (2, 4, 2, Isosceles), (1, 3, 4, Scalene).",
```

Comment:
Since 10, 10, 0 is a line and 0, 0, 0 is a point, and the discussion happened where we are not including the fringe of degenerate triangles, they should not be here.